src/App.vue

```diff
@@ -1,32 +1,81 @@
 <template>
-  <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
-    <router-view/>
-  </div>
-</template>
+  <v-app>
+    <v-app-bar app dark>
+      <v-toolbar-title>Localizar</v-toolbar-title>
 
-<style>
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-}
+      <v-spacer></v-spacer>
 
-#nav {
-  padding: 30px;
-}
+      <v-btn to="/" class="nav-btn">Home</v-btn>
+      <v-btn to="/about" class="nav-btn">About</v-btn>
 
-#nav a {
-  font-weight: bold;
-  color: #2c3e50;
-}
+      <v-spacer></v-spacer>
+
+      <v-btn to="/projects" class="nav-btn" v-if="authViewsVisible">
+        Projects
+      </v-btn>
 
-#nav a.router-link-exact-active {
-  color: #42b983;
+      <v-btn
+        color="error"
+        class="nav-btn"
+        v-if="authViewsVisible"
+        @click="logout"
+      >
+        Log out
+      </v-btn>
+
+      <v-btn
+        to="/sign_up"
+        color="success"
+        class="nav-btn"
+        v-if="!authViewsVisible"
+      >
+        Sign up
+      </v-btn>
+      <v-btn to="/login" color="info" class="nav-btn" v-if="!authViewsVisible">
+        Log in
+      </v-btn>
+    </v-app-bar>
+
+    <v-main>
+      <v-container fluid>
+        <router-view></router-view>
+      </v-container>
+    </v-main>
+  </v-app>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      authViewsVisible: false,
+    };
+  },
+  created() {
```

Comment:
tego nie ogarniam
wydaje mi się, że można to zrobić z wykorzystaniem mapGetters, sprawdź tutaj https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper
https://blog.logrocket.com/a-complete-guide-to-mapping-in-vuex/